superset-frontend/src/views/CRUD/data/dataset/AddDataset/AddDataset.test.tsx

```diff
@@ -17,17 +17,28 @@
  * under the License.
  */
 import React from 'react';
-import { render, screen } from 'spec/helpers/testing-library';
+import { render, screen, act } from 'spec/helpers/testing-library';
 import AddDataset from 'src/views/CRUD/data/dataset/AddDataset';
 
 describe('AddDataset', () => {
-  it('renders a blank state AddDataset', () => {
-    render(<AddDataset />);
+  const renderAndWait = async () => {
```

Comment:
If the purpose here to wait until the component has finished loading, you can use something like this

```
const waitForRender = (overrides: Record<string, any> = {}) =>
  waitFor(() => render(<AddDataset />);

await waitForRender();
```

Reply:
Oh yeah that's a lot cleaner! Fixed in [`this commit`](https://github.com/apache/superset/pull/21189/commits/890af00638acaa42d9f7d35bd86f3ad573739122).